lib/wasi-types/src/wasi/bindings_manual.rs

```diff
@@ -391,3 +391,21 @@ unsafe impl wasmer::FromToNativeWasmType for JoinFlags {
         false
     }
 }
+
+// TODO: if necessary, must be implemented in wit-bindgen
+unsafe impl wasmer::FromToNativeWasmType for DlFlags {
+    type Native = i32;
+
+    fn to_native(self) -> Self::Native {
+        self.bits() as i32
+    }
+
+    fn from_native(n: Self::Native) -> Self {
+        Self::from_bits_truncate(n as u32)
+    }
+
+    fn is_from_store(&self, _store: &impl wasmer::AsStoreRef) -> bool {
+        // TODO: find correct implementation
+        false
```

Comment:
We probably want to return true here

Reply:
That's how it's done for every one of these. I just copy-paste the existing definitions. Not sure if that function is being called at all though, or what the consequences of returning true here would be. Let sleeping dogs lie?